packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -86,14 +90,21 @@ export class EthereumConnection implements BcpConnection {
     return new EthereumConnection(baseUrl, chainId, options);
   }
 
+  private readonly blockPollIntervalMs: number;
+  private readonly accountPollIntervalMs: number;
   private readonly rpcClient: HttpJsonRpcClient;
   private readonly myChainId: ChainId;
   private readonly socket: StreamingSocket | undefined;
   private readonly scraperApiUrl: string | undefined;
-  private readonly erc20Tokens: ReadonlyMap<TokenTicker, Erc20>;
+  private readonly erc20Tokens: ReadonlyMap<TokenTicker, Erc20Options>;
+  private readonly erc20ContractReaders: ReadonlyMap<TokenTicker, Erc20>;
   private readonly codec: EthereumCodec;
 
   constructor(baseUrl: string, chainId: ChainId, options?: EthereumConnectionOptions) {
+    this.blockPollIntervalMs =
+      options && options.blockPollInterval ? options.blockPollInterval * 1000 : 4_000;
+    this.accountPollIntervalMs =
+      options && options.accountPollInterval ? options.accountPollInterval * 1000 : 5_000;
```

Comment:
Why would these be different?

Reply:
Not strictly required. They just used different values before. But it does not matter much. Should we reduce to one? I don't mind

Reply:
It doesn't seem important, but since they both have essentially the same granularity (block time) it seems strange to me that they wouldn't have the same default interval.

Reply:
Jap, let's simplify. With "poll interval" only we save a couple of lines